### DISCUSSION TITLE
Logo

### DISCUSSION THREAD

**Discussion by carlocorradini** [OpenRC/openrc#578] [Q&A]
Sorry if this is a silly question, but I couldn't come up with anything.
Is there a logo for OpenRC (like [Systemd](https://en.wikipedia.org/wiki/Systemd#/media/File:Systemd-logo.svg0)) or something similar?
Thanks 🥳

**Comment by navi-desu** [comment]
i've been wanting a proper logo though i am not really a graphic designer
